components/DocumentList.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import type {
+  DocumentListIAllDocumentModels,
+  IAllDocumentModels,
+} from "@bcc-code/bmm-sdk-fetch";
+
+const props = defineProps<{
+  list: DocumentListIAllDocumentModels | null;
+  pending: boolean;
+}>();
+
+watch(
+  props,
+  () => {
+    toolbarTitleStore().setToolbarTitle(props.list?.title ?? " ");
```

Comment:
Do all pages who use this for listing set this as title?

Reply:
Not sure what you mean. If the server returns a DocumentList it also provides the title.

Reply:
Well... I'd rather put these lines into each page... For me, the title is a part of the page and should not be controlled by a component. By this, you can use this component at more pages.